### PR TITLE
maint: update authors and mailmap

### PR DESCRIPTION
**Pull request by oscarbenjamin**
#### Brief description of what is fixed or changed

maint: update authors and mailmap

#### Other comments

NO ENTRY

### CODE REVIEW

**Comment by oscarbenjamin** [comment]
I will now contact these people to check that they are happy with the names used.

File: .mailmap [hunk 1]
```diff
@@ -493,6 +493,7 @@ Mohit Balwani <mohitbalwani.ict17@gmail.com> Mohitbalwani26 <mohitbalwani.ict17@
 Mathias Louboutin <mathias.louboutin@gmail.com>
 Psycho-Pirate <prakharsaxena.civ18@iitbhu.ac.in> Psycho-Pirate <prakharsaxena.cer18@itbhu.ac.in>
 Psycho-Pirate <prakharsaxena.civ18@iitbhu.ac.in> Psycho-Pirate <50932406+Psycho-Pirate@users.noreply.github.com>
+Psycho-Pirate <prakharsaxena.civ18@iitbhu.ac.in> Prakhar Saxena <prakharsaxena.civ18@iitbhu.ac.in>
```
Comment (oscarbenjamin) [inline .mailmap:496]: I've emailed to ask which name is preferred.

File: .mailmap [hunk 2]
```diff
@@ -548,3 +550,13 @@ Muhammed Abdul Quadir Owais <quadirowais200@gmail.com> MaqOwais <quadirowais200@
 Vaibhav Bhat <vaibhav.bhat2097@gmail.com> VBhat97 <vaibhav.bhat2097@gmail.com>
 Bhaskar Joshi <bhaskar.joshi@research.iiit.ac.in> BhaskarJoshi-01 <bhaskar.joshi@research.iiit.ac.in>
 Miguel Torres Costa <miguelptcosta1995@gmail.com>
+Elisha Hollander <just4now666666@gmail.com> donno2048 <just4now666666@gmail.com>
+Mamidi Ratna Praneeth <praneethratna@gmail.com> praneeth <praneethratna@gmail.com>
+Zach Carmichael <20629897+craymichael@users.noreply.github.com> Zach <20629897+craymichael@users.noreply.github.com>
+Anup Parikh <parikhanupk@gmail.com> Anup Parikh <84572003+parikhanupk@users.noreply.github.com>
+Harshit Yadav <harshityadav2k@gmail.com> Harshit Yadav <45384915+hyadav2k@users.noreply.github.com>
+Megan Ly <megan@artcompiler.com> Megan Ly <meganly@MacBook-Pro.local>
+Robert Pollak <robert.pollak@jku.at> Robert Pollak <robert.pollak@posteo.net>
```
Comment (oscarbenjamin) [inline .mailmap:559]: I have emailed to ask which of these addresses to use

File: .mailmap [hunk 3]
```diff
@@ -548,3 +550,13 @@ Muhammed Abdul Quadir Owais <quadirowais200@gmail.com> MaqOwais <quadirowais200@
...
+gum3ng <gum3ng@protonmail.com> gum3ng <78410304+gum3ng@users.noreply.github.com>
```
Comment (oscarbenjamin) [inline .mailmap:560]: I have emailed asking what name this contributor would like to have listed

File: .mailmap [hunk 4]
```diff
@@ -548,3 +550,13 @@ Muhammed Abdul Quadir Owais <quadirowais200@gmail.com> MaqOwais <quadirowais200@
...
+mostlyaman <amansharma110603@gmail.com> mostlyaman <85072876+mostlyaman@users.noreply.github.com>
```
Comment (oscarbenjamin) [inline .mailmap:561]: I have emailed asking what name to use

File: .mailmap [hunk 5]
```diff
@@ -548,3 +550,13 @@ Muhammed Abdul Quadir Owais <quadirowais200@gmail.com> MaqOwais <quadirowais200@
...
+robinechuca <raisin@ecomail.fr> robinechuca <61911191+robinechuca@users.noreply.github.com>
```
Comment (oscarbenjamin) [inline .mailmap:562]: I have emailed asking what name to use

File: .mailmap (hunk 1 above)
Comment (oscarbenjamin) [reply .mailmap:496]: The name "Psycho-Pirate" is preferred

File: AUTHORS [hunk 6]
```diff
@@ -1050,3 +1050,50 @@ Rodrigo Luger <rodluger@gmail.com>
 Marco Antônio Habitzreuter <mahabitzreuter@gmail.com>
 Ayush Bisht <bisht.ayush2001@gmail.com>
 Akshansh Bhatt <akshansh@tuta.io>
+Brandon T. Willard <brandonwillard@users.noreply.github.com>
+Thomas Aarholt <thomasaarholt@gmail.com>
+Hiren Chalodiya <hirenchalodiya99@gmail.com>
+Roland Dixon <rols121@gmail.com>
+dimasvq <dimas.vq.2020@bristol.ac.uk>
```
Comment (oscarbenjamin) [inline AUTHORS:1057]: I have emailed asking what name to use

File: AUTHORS [hunk 7]
```diff
@@ -1050,3 +1050,50 @@ Rodrigo Luger <rodluger@gmail.com>
...
+Sagar231 <sagarfeb298@gmail.com>
```
Comment (oscarbenjamin) [inline AUTHORS:1058]: I've emailed asking what name to use

File: AUTHORS [hunk 8]
```diff
@@ -1050,3 +1050,50 @@ Rodrigo Luger <rodluger@gmail.com>
...
+Michael Chu <michael02chu@gmail.com>
+Abby Ng <abigailjng@gmail.com>
+Angad Sandhu <55819847+angadsinghsandhu@users.noreply.github.com>
+Alexander Cockburn <alexander_cockburn12@hotmail.com>
+Yaser <yaseralosh@outlook.com>
```
Comment (oscarbenjamin) [inline AUTHORS:1063]: I've emailed asking what name to use

File: AUTHORS [hunk 9]
```diff
@@ -1050,3 +1050,50 @@ Rodrigo Luger <rodluger@gmail.com>
...
+Davide_sd <sandona.davide@gmail.com>
```
Comment (oscarbenjamin) [inline AUTHORS:1064]: I've emailed asking what name to use

File: AUTHORS [hunk 10]
```diff
@@ -1050,3 +1050,50 @@ Rodrigo Luger <rodluger@gmail.com>
...
+gutow <gutow@uwosh.edu>
```
Comment (oscarbenjamin) [inline AUTHORS:1065]: I've emailed asking what name to use

File: AUTHORS [hunk 11]
```diff
@@ -1050,3 +1050,50 @@ Rodrigo Luger <rodluger@gmail.com>
...
+Nihir Agarwal <f20180701@pilani.bits-pilani.ac.in>
+Lee Johnston <lee.johnston.100@gmail.com>
+Zach Carmichael <20629897+craymichael@users.noreply.github.com>
+vijair <vijair@umich.edu>
+vijairam20 <vijairamg@gmail.com>
```
Comment (oscarbenjamin) [inline AUTHORS:1070]: I've emailed both addresses to see if this is the same person

File: AUTHORS [hunk 12]
```diff
@@ -1050,3 +1050,50 @@ Rodrigo Luger <rodluger@gmail.com>
...
+dsaw <devsaw115@gmail.com>
```
Comment (oscarbenjamin) [inline AUTHORS:1071]: I've emailed asking what name to use

File: AUTHORS [hunk 13]
```diff
@@ -1050,3 +1050,50 @@ Rodrigo Luger <rodluger@gmail.com>
...
+Hanspeter Schmid <hanspeter.schmid@fhnw.ch>
+oostben <oostben@umich.edu>
```
Comment (oscarbenjamin) [inline AUTHORS:1073]: I've emailed asking what name to use

File: AUTHORS [hunk 14]
```diff
@@ -1050,3 +1050,50 @@ Rodrigo Luger <rodluger@gmail.com>
...
+Nikita <nikita.student.cse19@itbhu.ac.in>
+Aman <amanmourya295@gmail.com>
```
Comment (oscarbenjamin) [inline AUTHORS:1075]: I've emailed asking what name to use

File: AUTHORS [hunk 15]
```diff
@@ -1050,3 +1050,50 @@ Rodrigo Luger <rodluger@gmail.com>
...
+gutow <gutow@uwosh.edu>
+Nihir Agarwal <f20180701@pilani.bits-pilani.ac.in>
+Lee Johnston <lee.johnston.100@gmail.com>
+Zach Carmichael <20629897+craymichael@users.noreply.github.com>
+vijair <vijair@umich.edu>
+vijairam20 <vijairamg@gmail.com>
+dsaw <devsaw115@gmail.com>
+Hanspeter Schmid <hanspeter.schmid@fhnw.ch>
+oostben <oostben@umich.edu>
+Nikita <nikita.student.cse19@itbhu.ac.in>
```
Comment (oscarbenjamin) [inline AUTHORS:1074]: I've emailed asking what name to use

File: AUTHORS [hunk 16]
```diff
@@ -1050,3 +1050,50 @@ Rodrigo Luger <rodluger@gmail.com>
...
+Aman <amanmourya295@gmail.com>
+Shashank KS <shashankks@sahaj.ai>
```
Comment (oscarbenjamin) [inline AUTHORS:1076]: I've emailed asking what name to use

File: .mailmap (hunk 2 above)
Comment (oscarbenjamin) [reply .mailmap]: The posteo.net address should be used

File: AUTHORS (hunk 16 above)
Comment (oscarbenjamin) [reply AUTHORS:1076]: This email gave address not found...

Comment (oscarbenjamin) [reply AUTHORS:1076]: @shashankks0987 this entry in the authors file seems to come from your commits.

I tried emailing the address but it doesn't seem to be a valid email address.

What name and email address would you like to have recorded for your contributions?

Currently the name is listed as "Shashank KS" but I can change this to list your full name if you prefer.

File: AUTHORS [hunk 17]
```diff
@@ -1050,3 +1050,50 @@ Rodrigo Luger <rodluger@gmail.com>
...
+mostlyaman <amansharma110603@gmail.com>
+Anup Parikh <parikhanupk@gmail.com>
+LucyMountain <lucymountain1@icloud.com>
```
Comment (oscarbenjamin) [inline AUTHORS:1079]: I've emailed asking what name to use

File: AUTHORS [hunk 18]
```diff
@@ -1050,3 +1050,50 @@ Rodrigo Luger <rodluger@gmail.com>
...
+Miguel Torres Costa <miguelptcosta1995@gmail.com>
+Rikard Nordgren <rikard.nordgren@farmaci.uu.se>
+Arun sanganal <74652697+ArunSanganal@users.noreply.github.com>
+Kamlesh Joshi <72374645+kamleshjoshi8102@users.noreply.github.com>
+Joseph Rance <56409230+Joseph-Rance@users.noreply.github.com>
+Huangxiaodui <hdrong.42@163.com>
```
Comment (oscarbenjamin) [inline AUTHORS:1085]: I've emailed asking what name to use

File: AUTHORS [hunk 19]
```diff
@@ -1050,3 +1050,50 @@ Rodrigo Luger <rodluger@gmail.com>
...
+Nils Schulte <47043622+Schnilz@users.noreply.github.com>
+Matt Bogosian <matt@bogosian.net>
+Elisha Hollander <just4now666666@gmail.com>
+InfProbSciX <infprobscix@gmail.com>
```
Comment (oscarbenjamin) [inline AUTHORS:1089]: I've emailed asking what name to use

File: AUTHORS [hunk 20]
```diff
@@ -1050,3 +1050,50 @@ Rodrigo Luger <rodluger@gmail.com>
...
+Mamidi Ratna Praneeth <praneethratna@gmail.com>
+Jeffrey Ryan <jeffaryan7@gmail.com>
+Jonathan Daniel <36337649+jond01@users.noreply.github.com>
+robinechuca <raisin@ecomail.fr>
+Nornort <francois.lalleve@laposte.net>
```
Comment (oscarbenjamin) [inline AUTHORS:1094]: I've emailed asking what name to use

File: AUTHORS [hunk 21]
```diff
@@ -1050,3 +1050,50 @@ Rodrigo Luger <rodluger@gmail.com>
...
+gautammenghani <gautammenghani14@gmail.com>
```
Comment (oscarbenjamin) [inline AUTHORS:1095]: I've emailed asking what name to use

File: AUTHORS [hunk 22]
```diff
@@ -1050,3 +1050,50 @@ Rodrigo Luger <rodluger@gmail.com>
...
+gum3ng <gum3ng@protonmail.com>
+Remco de Boer <redeboer@gmx.com>
+sebastian-east <sebastianeast@ymail.com>
```
Comment (oscarbenjamin) [inline AUTHORS:1098]: I've emailed asking what name to use

File: AUTHORS [hunk 23]
```diff
@@ -1050,3 +1050,50 @@ Rodrigo Luger <rodluger@gmail.com>
...
+evani balasubramanyam <balasubramanyam.evani@gmail.com>
```
Comment (oscarbenjamin) [inline AUTHORS:1099]: I've emailed asking what name to use

File: .mailmap [hunk 24]
```diff
@@ -548,3 +550,14 @@ Muhammed Abdul Quadir Owais <quadirowais200@gmail.com> MaqOwais <quadirowais200@
...
+Mamidi Ratna Praneeth <praneethratna@gmail.com> praneeth <praneethratna@gmail.com>
+Zach Carmichael <20629897+craymichael@users.noreply.github.com> Zach <20629897+craymichael@users.noreply.github.com>
+Anup Parikh <parikhanupk@gmail.com> Anup Parikh <84572003+parikhanupk@users.noreply.github.com>
+Harshit Yadav <harshityadav2k@gmail.com> Harshit Yadav <45384915+hyadav2k@users.noreply.github.com>
+Megan Ly <megan@artcompiler.com> Megan Ly <meganly@MacBook-Pro.local>
+Robert Pollak <robert.pollak@posteo.net> Robert Pollak <robert.pollak@jku.at>
+gum3ng <gum3ng@protonmail.com> gum3ng <78410304+gum3ng@users.noreply.github.com>
+mostlyaman <amansharma110603@gmail.com> mostlyaman <85072876+mostlyaman@users.noreply.github.com>
+robinechuca <raisin@ecomail.fr> robinechuca <61911191+robinechuca@users.noreply.github.com>
+Huangduirong <huangduirong@huawei.com> Huangxiaodui <hdrong.42@163.com>
```
Comment (oscarbenjamin) [inline .mailmap:563]: I added this name/email after an email exchange with the recorded email address.

File: AUTHORS (hunk 10 above)
Comment (gutow) [reply AUTHORS:1065]: I replied to your e-mail, but will also put it here. Please use `Jonathan Gutow <gutow@uwosh.edu>`. Thanks.

File: .mailmap (hunk 4 above)
Comment (mostlyaman) [reply .mailmap:561]: Hi, I've replied to your mail. Please use Aman Sharma as my name. Thanks.

File: AUTHORS (hunk 16 above)
Comment (shashankks0987) [reply AUTHORS:1076]: @oscarbenjamin
```
 Name: Shashank KS ,
 Email: shashankks0987@gmail.com 
```
I'd like this email address and name to be associated with my contributions

File: AUTHORS (hunk 23 above)
Comment (BalasubramanyamEvani) [reply AUTHORS:1099]: I have replied to your mail. I'm putting it here as well. Please use `Evani Balasubramanyam <balasubramanyam.evani@gmail.com>`. Thank you!

File: AUTHORS (hunk 15 above)
Comment (oscarbenjamin) [reply AUTHORS:1074]: Just "Nikita" is fine

File: AUTHORS (hunk 10 above)
Comment (oscarbenjamin) [reply AUTHORS:1065]: This should be
```
Jonathan Gutow <gutow@uwosh.edu>
```

File: AUTHORS (hunk 13 above)
Comment (oscarbenjamin) [reply AUTHORS:1073]: This should be
```
Ben Oostendorp <oostben@umich.edu>
```

File: .mailmap (hunk 3 above)
Comment (oscarbenjamin) [reply .mailmap:560]: This is
```
Gautam Menghani <gum3ng@protonmail.com>
```

File: .mailmap (hunk 4 above)
Comment (oscarbenjamin) [reply .mailmap:561]: This is
```
Aman Sharma <amansharma110603@gmail.com>
```

File: AUTHORS (hunk 23 above)
Comment (oscarbenjamin) [reply AUTHORS:1099]: Name is "Evani Balasubramanyam"

File: AUTHORS (hunk 22 above)
Comment (oscarbenjamin) [reply AUTHORS:1098]: Sebastian East <sebastianeast@ymail.com>

File: .mailmap (hunk 5 above)
Comment (oscarbenjamin) [reply .mailmap:562]: Robin RICHARD

File: AUTHORS (hunk 16 above)
Comment (oscarbenjamin) [reply AUTHORS:1076]: Thanks, I've added that.

File: AUTHORS (hunk 9 above)
Comment (oscarbenjamin) [reply AUTHORS]:
```
Davide Sandonà <sandona.davide@gmail.com>
```

File: AUTHORS (hunk 11 above)
Comment (oscarbenjamin) [reply AUTHORS]: Both addresses the same. Name is
```
Vijairam Ganesh Moorthy <vijairamg@gmail.com>
```

File: AUTHORS (hunk 18 above)
Comment (oscarbenjamin) [reply AUTHORS]:
```
Huangduirong <huangduirong@huawei.com>
```

File: AUTHORS (hunk 21 above)
Comment (oscarbenjamin) [reply AUTHORS]:
```
Gautam Menghani <gum3ng@protonmail.com>
```

File: AUTHORS (hunk 12 above)
Comment (oscarbenjamin) [reply AUTHORS:1070]: This is the same as `Devesh Sawant <devesh47cool@gmail.com>`

File: AUTHORS (hunk 6 above)
Comment (oscarbenjamin) [reply AUTHORS:1057]: @dimasvq what name and email address should be used here?

File: AUTHORS (hunk 7 above)
Comment (oscarbenjamin) [reply AUTHORS:1058]: @Sagar231 what name and email should be used here?

File: AUTHORS (hunk 8 above)
Comment (oscarbenjamin) [reply AUTHORS:1063]: @YaserAlOsh what name and email should be recorded here?

File: AUTHORS (hunk 14 above)
Comment (oscarbenjamin) [reply AUTHORS:1073]: @hackman01 what name and email should be used here?

File: AUTHORS (hunk 19 above)
Comment (oscarbenjamin) [reply AUTHORS:1087]: @InfProbSciX what name and email should be used here?

File: AUTHORS (hunk 20 above)
Comment (oscarbenjamin) [reply AUTHORS:1092]: @robinechuca is Nornort the same person as robinechuca? Which name and email should be used?

Comment (oscarbenjamin) [reply AUTHORS]: I've set Nornort as another alias for Robert Richard

File: AUTHORS [hunk 25]
```diff
@@ -1050,3 +1050,50 @@ Rodrigo Luger <rodluger@gmail.com>
...
+InfProbSciX <infprobscix@gmail.com>
+Mamidi Ratna Praneeth <praneethratna@gmail.com>
+Jeffrey Ryan <jeffaryan7@gmail.com>
+Jonathan Daniel <36337649+jond01@users.noreply.github.com>
+Robin Richard <raisin@ecomail.fr>
+Gautam Menghani <gum3ng@protonmail.com>
+Remco de Boer <redeboer@gmx.com>
+Sebastian East <sebastianeast@ymail.com>
+Evani Balasubramanyam <balasubramanyam.evani@gmail.com>
+rprkh <r.parikh@somaiya.edu>
```
Comment (oscarbenjamin) [inline AUTHORS:1096]: @rprkh what name and email address should I use here?

File: AUTHORS [hunk 26]
```diff
@@ -1050,3 +1050,50 @@ Rodrigo Luger <rodluger@gmail.com>
...
+Jason Ross <jasonross1024@gmail.com>
+Akshansh <53227127+akshanshbhatt@users.noreply.github.com>
```
Comment (oscarbenjamin) [inline AUTHORS:1098]: @akshanshbhatt should I record this as an alias for 
```
Akshansh Bhatt <akshansh@tuta.io>
```
?

File: AUTHORS [hunk 27]
```diff
@@ -1050,3 +1050,50 @@ Rodrigo Luger <rodluger@gmail.com>
...
+nanjekyejoannah <joannah.nanjekye@ibm.com>
```
Comment (oscarbenjamin) [inline AUTHORS:1099]: @nanjekyejoannah what name and email address should I use here?

Comment (oscarbenjamin) [reply AUTHORS:1099]: Maybe:
```
Joannah Nanjekye <joannah.nanjekye@ibm.com>
```

File: AUTHORS (hunk 26 above)
Comment (akshanshbhatt) [reply AUTHORS:1098]: Yes @oscarbenjamin. Thanks.

File: AUTHORS (hunk 25 above)
Comment (rprkh) [reply AUTHORS:1096]: Rahil Parikh <r.parikh@somaiya.edu>

File: AUTHORS (hunk 26 above)
Comment (oscarbenjamin) [reply AUTHORS]: Thanks I've added that.

File: AUTHORS (hunk 25 above)
Comment (oscarbenjamin) [reply AUTHORS]: Thanks. I've added that.

File: AUTHORS (hunk 19 above)
Comment (oscarbenjamin) [reply AUTHORS]: This is
```
 Aditya Ravuri <infprobscix@gmail.com>
```

File: AUTHORS (hunk 17 above)
Comment (oscarbenjamin) [reply AUTHORS]: This is
```
Lucy Mountain <lucymountain1@icloud.com>
```

**Comment by YaserAlOsh** [comment]
Hi there, thanks for contacting me.
Yaser AlOsh for the name.
The same email is fine.

File: AUTHORS (hunk 8 above)
Comment (oscarbenjamin) [reply AUTHORS:1063]: This is
```
Yaser AlOsh <yaseralosh@outlook.com>
```

**Comment by oscarbenjamin** [comment]
> Yaser AlOsh for the name.

Thanks for responding. I've added that.

File: AUTHORS (hunk 27 above)
Comment (oscarbenjamin) [reply AUTHORS]: > Maybe:
> 
> ```
> Joannah Nanjekye <joannah.nanjekye@ibm.com>
> ```

That's what I've changed it to.

**Pull request merged**